### PR TITLE
Dependabot front end PR bulk merge

### PR DESCRIPTION
**Pull request by BrendanBeachBC**
# Description

This PR includes the following proposed change(s):

The following Dependabot PRs:
https://github.com/bcgov/jag-file-submission/pull/1129
https://github.com/bcgov/jag-file-submission/pull/1130
https://github.com/bcgov/jag-file-submission/pull/1131
https://github.com/bcgov/jag-file-submission/pull/1132
https://github.com/bcgov/jag-file-submission/pull/1133
https://github.com/bcgov/jag-file-submission/pull/1134
https://github.com/bcgov/jag-file-submission/pull/1135
https://github.com/bcgov/jag-file-submission/pull/1136
https://github.com/bcgov/jag-file-submission/pull/1137

Also updated snapshot tests

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Refactoring / Documentation
- [ ] Version change

if your change is a breaking change, please add `breaking change` label to this PR

## How Has This Been Tested?

Manual tests, unit tests, storybook.

## Does the change impact or break the Docker build?

- [ ] Yes
- [x] No

If Yes: Has Docker been updated accordingly?

- [ ] Yes
- [ ] No

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by alexjoybc** [comment]
There is a failure in the Javascript test, could you debug @BrendanBeachBC ?

**Comment by BrendanBeachBC** [comment]
Javascript tests were failing due to linting errors, which have now been fixed.

File: src/frontend/efiling-frontend/src/modules/helpers/creditCardAlerts.js [hunk 1]
```diff
@@ -23,93 +23,86 @@ const registerCard = () => {
...
-          >
-            Register a new Credit Card.
-          </span>
-        </p>
-      }
-    />
-  );
-};
+const failedUpdateCreditCard = () => (
+  <Alert
```
Comment (akroon3r) [inline src/frontend/efiling-frontend/src/modules/helpers/creditCardAlerts.js:81]: I'm interested in the removal of the return. Give me a little insight?

Comment (BrendanBeachBC) [reply src/frontend/efiling-frontend/src/modules/helpers/creditCardAlerts.js:81]: This was something that was caught by eslint after updating. When using Javascript arrow notation for functions, if your function is nothing but a return statement, you can eliminate the "return" and just return the expression directly. 

`const myFunction = () => {return (<div>Hello</div>)}`

is the same as doing

`const myFunction = () =>  (<div>Hello</div>)`

https://stackoverflow.com/a/52637049

**Review by akroon3r — APPROVED** [review]

**Pull request merged**